### ISSUE TITLE
Getting "Xpra server is not installed"

### ISSUE THREAD

**Issue by midiwidi** [Xpra-org/xpra#3833]
**Describe the bug**
Running `xpra start --start-child=xterm --bind-tcp=0.0.0.0:10000` results in
```
xpra initialization error:
 Xpra server is not installed
```

**To Reproduce**
1. Install Xpra as described at the very bottom of the Download page in the Wiki [here](https://github.com/Xpra-org/xpra/wiki/Download)
```
DISTRO=focal
#install https support for apt (which may be installed already):
sudo apt update
sudo apt install apt-transport-https software-properties-common
sudo apt install ca-certificates
# add xpra GPG key:
sudo wget -O "/usr/share/keyrings/xpra.asc" https://xpra.org/gpg.asc
# add the xpra repository:
sudo wget -O "/etc/apt/sources.list.d/xpra.sources" https://xpra.org/repos/$DISTRO/xpra.sources
# add the optional beta channel:
# wget -O "/etc/apt/sources.list.d/xpra-beta.sources" https://xpra.org/repos/$DISTRO/xpra-beta.sources
# install the xpra package:
sudo apt update
sudo apt install xpra
```
or build DEB packages from scratch as described [here](https://github.com/Xpra-org/xpra/blob/master/docs/Build/Debian.md)

2. Run `xpra start --start-child=xterm --bind-tcp=0.0.0.0:10000`

**System Information (please complete the following information):**
 - Server OS: Ubuntu 20.04.6 LTS (focal) running in VirtualBox 7.0 on a Windows 10 Host 
 - Client OS: N/A
 - Xpra Server Version: 
```
   $ xpra --version
      xpra v5.0-r33131 (gd1c801062) beta
```
 - Xpra Client Version: N/A
 - Other Server Software Versions which might be relevant:
```
    $ python --version
    Python 3.11.3
    $ pip --version
    pip 23.1 from /home/midiwidi/.local/lib/python3.11/site-packages/pip (python 3.11)
```

**Additional context**
My goal is to access a python tkinter application remotely from other computers via the web browser. 
I tried both, installing Xpra from pre-compiled packages and also compiled it myself, but the result was the same. When I run `xpra start ...` I get an error that the Xpra server is not installed. I also tried to specifically install the server component using `sudo apt --no-install-recommends install xpra-server`, but this package is unknown.
I'm not sure if that is connected or a totally different problem, but if I run `xpra list` or `xpra showconfig` it throws the exception
```
xpra main error:
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/xpra/scripts/main.py", line 153, in main
    return run_mode(script_file, cmdline, err, options, args, mode, defaults)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/xpra/scripts/main.py", line 453, in run_mode
    set_name("Xpra", "Xpra %s" % mode.strip("_"))
  File "/usr/local/lib/python3.11/dist-packages/xpra/platform/__init__.py", line 83, in set_name
    set_prgname(prgname or _prgname)
  File "/usr/local/lib/python3.11/dist-packages/xpra/platform/__init__.py", line 90, in set_prgname
    from gi.repository import GLib  # @UnresolvedImport
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 42, in <module>
    from . import _gi
ImportError: cannot import name '_gi' from partially initialized module 'gi' (most likely due to a circular import) (/usr/lib/python3/dist-packages/gi/__init__.py)
```

How can I get the server running?

**Comment by totaam** [comment]
> ```shell
> $ python --version
>     Python 3.11.3
> ```

Whatever you've done to get this Python version installed is the problem.
The default version shipped with `Ubuntu 20.04` is `3.8.10`.

You may be able to workaround the issue by invoking xpra with the correct version of Python explicitly:
```shell
python3.8 /usr/bin/xpra ...
```
But eventually you will hit other issues.

This is not a supported configuration. `/usr/bin/python3` should be symlink to `python3.8`.

**Comment by midiwidi** [comment]
Thank you so much for this hint. I changed the python back to 3.8
```
$ sudo update-alternatives --config python3
There are 2 choices for the alternative python3 (providing /usr/bin/python3).

  Selection    Path                 Priority   Status
------------------------------------------------------------
  0            /usr/bin/python3.11   2         auto mode
  1            /usr/bin/python3.11   2         manual mode
* 2            /usr/bin/python3.8    1         manual mode

Press <enter> to keep the current choice[*], or type selection number: 2
```

and now it works beautifully.
